nanome_docking/menus/DockingMenu.py

```diff
@@ -429,6 +423,9 @@ def __init__(self, plugin):
         self._exhaust_slider = menu_root.find_node("ExhaustSlider").get_content()
         self._visual_scores = False
 
+        self._btn_deterministic_results = menu_root.find_node("btn_deterministic_results").get_content()
+        self._btn_deterministic_results.register_pressed_callback(self.btn_determinisitc_results_pressed_callback)
```

Comment:
You could eliminate this callback by setting `self._btn_deterministic_results.toggle_on_press = True`